datasets/us/fincen_msb/crawler.py

```diff
@@ -0,0 +1,47 @@
+from csv import DictReader
+from typing import Dict, List
+from rigour.mime.types import CSV
+
+from zavod import Context
+from zavod import helpers as h
+
+
+def crawl_row(context: Context, row: Dict[str, List[str]]):
+    name = row.pop("LEGAL NAME")
+    street = row.pop("STREET ADDRESS")
+    city = row.pop("CITY")
+    state = row.pop("STATE")
+    zip_code = row.pop("ZIP")
```

Comment:
nitpick: this can be popped as you build the address, making things a touch more concise